### DISCUSSION TITLE
Install the Authgear Web SDK

### DISCUSSION THREAD

**Discussion by findnestsg** [authgear/authgear-server#2720] [Get Help] [answered]
@fungc-io  

experiencing an Issue while installing the packages mentioned on https://docs.authgear.com/get-started/website 
pl troubleshoot

**Comment by fungc-io** [comment]
Hi @findnestsg 

Which package is it? Is it `@authgear/web`?

Can you provide the error message or screenshot of the problem you've experienced?

**Reply by findnestsg** [reply]
yes error package @authgear/web
please see error and assist thanks 
(https://user-images.githubusercontent.com/120375935/207038625-18324925-4c55-407f-a6ba-95ae4706e87f.jpeg)

**Reply by fungc-io** [reply]
Hi @findnestsg 
From what I see from the error message is that `yarn` is not properly installed on your computer. 
You can use `npm` or `yarn` as the package manager. Can you check if you have either installed?

**Reply by findnestsg** [reply]
thanks 
npm is installed in my system
(https://user-images.githubusercontent.com/120375935/207040555-2822bb17-7b14-42bc-9714-58335274630e.jpeg)

**Reply by fungc-io** [reply, marked as answer]
Can you try installing the package using the npm command?

`npm install --save --save-exact @authgear/web`

**Reply by findnestsg** [reply]
thanks 
updating my node and npm first

**Reply by findnestsg** [reply]
seems it is resolved 
Package installed issue was with node registry
